zktest/src/archive/test2/B70-ZK-2603.zul

```diff
@@ -118,11 +118,11 @@ Copyright (C)  Potix Corporation. All Rights Reserved.
 	</button>
 	<button label="attach">
 		<attribute name="onClick">
-			grid2.setPage(self.getPage());
 			gridmodel.clear();
 			gridmodel.add("abc");
 			gridmodel.add("abc");
 			gridmodel.add("abc");
+			grid2.setPage(self.getPage());
```

Comment:
does the setPage order effect?

Reply:
It is confirmed the zk.log order will be changed, but this shouldn't be an issue.
We just want to ensure when attached, list 1 and list 2 are both updated.
I have Charles refine the test case.